### PR TITLE
Added full_url field for image rendition

### LINKED ISSUE #7558: Use Full URL for ImageRenditionField.
### Is your proposal related to a problem?

I'm a big fan of the new `full_url` field that images have and would like them to be easily used in the API.

Assuming one's frontend app is living on a different domain to the Wagtail API then the default relative URLs aren't as useful.

### Describe the solution you'd like

Add `full_url` to the output of `ImageRenditionField`.

I propose it just replace the `url` field altogether, but both could be returned.

### Describe alternatives you've considered

I've been extending the `ImageRenditionField` for use in my own projects

### Additional context

(Write your answer here.)

### PR DESCRIPTION
**Pull request by PaarthAgarwal**
I have added the `full_url` field for image renditions. Fixes #7558.
It is my first time doing API changes here, so please check and let me know if there are any necessary adjustments.

### CODE REVIEW

File: docs/advanced_topics/api/v2/configuration.rst [hunk 1]
```diff
@@ -238,7 +238,8 @@ This would add the following to the JSON:
             "height": 1125
         },
         "feed_image_thumbnail": {
-            "url": "/media/images/a_test_image.fill-100x100.jpg",
+            "url": "http://www.example.com/media/images/a_test_image.fill-100x100.jpg",
+            "full_url": "/media/images/a_test_image.fill-100x100.jpg",
```
Comment (lb-) [inline docs/advanced_topics/api/v2/configuration.rst:242]: Should these values be the other way around? The `url` should be the one without https (i.e. current behaviour)

**Review by lb- — CHANGES_REQUESTED** [review]
Looks pretty good. Just one comment. 

I haven't run locally yet though.

File: docs/advanced_topics/api/v2/configuration.rst (hunk 1 above)
Comment (PaarthAgarwal) [reply docs/advanced_topics/api/v2/configuration.rst:242]: Sorry!

Comment (lb-) [reply docs/advanced_topics/api/v2/configuration.rst:242]: No need to be sorry, it's why we do reviews. It's super easy to miss things - another set of eyes always helps.

**Comment by INFINTY1999** [comment]
Hey, parth. I am Ashish I am new open source can you give your insta or your twitter so that I can ask you. if I am have any problem.

**Review by lb- — APPROVED** [review]
looks great, tested locally and works well
Screen Shot 2022-04-20 at 7 04 54 am
Screen Shot 2022-04-20 at 7 04 47 am

Will push up the changelog and a tweak to the documentation and then merge.

**Pull request merged**